src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -103,15 +104,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     noteEntity.setProperty("categories", note.classifyText());
     noteEntity.setProperty("timestamp", timestamp);
 
+    OutputStream outStream = new ByteArrayOutputStream();
     try {
-      note.writeConvertedDoc();
+      note.writeConvertedDoc(outStream);
     } catch (Docx4JException e) {
       System.out.println(e);
```

Comment:
While you're here, maybe change this to `e.printStackTrace()` to make future debugging easier?